tests/src/test/java/org/jboss/forge/test/roaster/model/util/MethodsTest.java

```diff
@@ -5,40 +5,32 @@
  * http://www.eclipse.org/legal/epl-v10.html
  */
 
-package org.jboss.forge.roaster.model.util;
+package org.jboss.forge.test.roaster.model.util;
 
 import static org.junit.Assert.assertArrayEquals;
-import static org.junit.Assert.assertEquals;
 
 import java.awt.List;
+import java.io.InputStream;
 import java.util.Date;
 import java.util.Map;
 
+import javax.swing.UIManager;
+
+import org.jboss.forge.roaster.model.util.Methods;
 import org.junit.Test;
 
 /**
- *
  * @author <a href="mailto:ggastald@redhat.com">George Gastaldi</a>
  */
 public class MethodsTest
 {
 
    @Test
-   public void testParamNameSingleUpperCase()
-   {
-      assertEquals("facesContext", Methods.toParamName("FacesContext"));
-   }
-
-   @Test
-   public void testParamNameMultipleUpperCase()
-   {
-      assertEquals("uiComponent", Methods.toParamName("UIComponent"));
-   }
-
-   @Test
-   public void testParamNameAllLowerCase()
+   public void testGenerateParametersNamesWithDifferentCases()
    {
-      assertEquals("myclass", Methods.toParamName("myclass"));
+      String[] paramNames = Methods
```

Comment:
This is not exactly testing the `Methods.toParamName` method.

Reply:
Yes, that is true. My personal developer option is, that only tests on the public API is needed, if the internal logic is not complex. That helps, to avoid unnecessary changes of tests if the internal details are changing.
In this case, the internal logic is not complex, and the method "toParamName" is indirectly called. 

If you think, we need a direct test, either the public the test needs to be in another package, the method needs to be public or my PR is not valid.

BR,
Kai

Reply:
Ah I haven't realized that was a private method. You're right.